dependencies.yaml

```
@@ -347,6 +347,7 @@ dependencies:
           - pytest-cov
           - pytest-xdist
           - geopandas>=0.11.0
+          - cuspatial==23.10.*
       - output_types: [requirements, pyproject]
         packages:
           - pyproj
```

```suggestion
          - pyproj>=3.6.0,<3.7a0
```

Apparently not specifying a loose pin causes conda to implicitly use a tight pin to the latest version. Doing this should avoid the problem of pinning to an older libtiff that proj 9.2 uses, and caused solver problems for us. Am I right @raydouglass ? See https://github.com/rapidsai/cuspatial/pull/1276